contracts/BnomialNFT.sol

```diff
@@ -62,17 +62,17 @@ contract BnomialNFT is ERC721, Ownable {
             );
         }
 
-        string memory part0 = '{"name":"Bnomial Badges",';
+        string memory part0 = "{\"name\":\"Bnomial Badges\",";
```

Comment:
I guess these changes come from some automatic formatting. Do you use `prettier`? I think we should add a standard config.

Reply:
No, I didn't use it. Okay, I will apply prettier.